_posts/2019-09-14-first-time-things-to-bring.md

```diff
@@ -0,0 +1,40 @@
+---
+author: Drew Winstel
+category: General
+date: 2019-09-14 06:00:00
+layout: post
+title: "It’s my first DjangoCon US. What should I bring?"
+---
+
+It’s your first DjangoCon US (or maybe even your first tech conference). Are you concerned about what you should pack?
+
+I (Drew Winstel, the 2019 Opportunity Grants chair) was in that position two years ago at DjangoCon US 2017 in Spokane, WA.  Here’s what I think you should pack.
+
+## Notepad and pen/pencil, or a laptop and charger
+Perhaps you like your notes a bit more analog? A notepad is exactly what you need. Many people like to leave their laptops in their rooms entirely during talks and take notes on paper instead. Plus, it’s great for logging contact info of people you meet if you don’t have the time or patience to log their contact info into your phone’s contacts immediately.
+
+If you can't live without your laptop, make sure you bring your charging cable. There’s ample Wi-Fi at the conference provided by the Marriott, so you’ll have the internet at your disposal. Many attendees will have their laptops open to take notes, follow along with slides, and/or test out the concepts being presented during talks. Plus, it’s much easier to live-tweet talks from your computer than your phone if you’re into that sort of thing.
+
+## Refillable water bottle
+It’s pretty annoying getting thirsty in the middle of a conference talk. The Marriott provides water dispensers in the hotel lobby, each of the conference rooms, the green room, and the Rio Vista Pavilion, so there are ample opportunities to keep yourself hydrated. Plus, you can be greener by reusing your own vessel instead of single-use disposable cups.
+
+## Comfortable clothes
```

Comment:
Could you add something about bringing a sweater in case the talk rooms are chilly?